components/List/index.js

```diff
@@ -0,0 +1,38 @@
+import { StyledCard } from "../Card/Card.styled";
+import { StyledSpan } from "../Card/Card.styled";
+import Image from "next/image";
+
+export default function List({ rides }) {
+  return (
+    <div>
+      {rides.map((ride) => (
+        <StyledCard key={ride.id}>
```

Comment:
When mapping over an array and create components for each item in the array, I would always use a list and list items: by doing so, the screen reader can tell how many items are going to be read and the user can skip etc.).

Please use a list here  ;)